src/content/Effects.js

```diff
@@ -222,7 +238,7 @@ Possible fixes:
                     <tr>
                         <td>The type of an <i>effect polymorphic</i> function from <Code>a</Code> to <Code>b</Code> with
                             effect <Code>ef1 and ef2</Code> (i.e. pure if both <Code>ef1</Code> and <Code>ef2</Code> are
-                            pure.)
+                            true.)
```

Comment:
If my explanation above is correct, then this should probably also use `true` instead of `pure`, because the value of individual effect variables does not necessarily have to mean pure or impure.

Reply:
Looks correct to me.